pyccel/stdlib/parallel/openmp.py

```diff
@@ -1,4 +1,10 @@
 # coding: utf-8
+# pylint: disable=missing-module-docstring
```

Comment:
Please do not disable missing docstrings outside of the tests/ folder.

I am happy for you to leave this file without a docstring

@saidctb should this file still exist?

Reply:
we can delet it